test-directory/tests.sh

```diff
@@ -1,3 +1,9 @@
 #!bin/sh
 emacs --batch --eval '(message (pp (+ 2 2)))'
 
+# test for Download last uploaded file 
+echo "let plus = ( + )">expected-demo.ml
+echo -ne '\ntest\n' | learn-ocaml-client --server=http://localhost --html --id=demo expected-demo.ml 
```

Comment:
Question: what is the purpose of this `'\ntest\n'` text sent to the stdin of learn-ocaml-client?

Reply:
It's the nickname associated to the token created when the first \n is interpreted.
We could add a docker volume to store tokens so we can use  some tokens in tests but i think this is simpler

Reply:
ok, let's keep it simple. So maybe you could just as well remove that volume declaration above?

`-v learn-ocaml-sync-demo:/sync`